packages/ckeditor5-engine/tests/conversion/upcastdispatcher.js

```diff
@@ -605,6 +605,39 @@ describe( 'UpcastDispatcher', () => {
 				sinon.assert.calledOnce( spy );
 			} );
 
+			it( 'should insert paragraph if it\'s allowed and if it would allow conversion', () => {
```

Comment:
Maybe, "should auto-paragraph element if it is not allowed at insert postion but is inserted where paragraph is allowed"?